app/src/main/java/com/microsoft/graph/connect/GraphServiceClientManager.java

```diff
@@ -70,9 +70,9 @@ public synchronized IGraphServiceClient getGraphServiceClient(IAuthenticationPro
             IClientConfig clientConfig = DefaultClientConfig.createWithAuthenticationProvider(
                     authenticationProvider
             );
-            mGraphServiceClient = new GraphServiceClient.Builder().fromConfig(clientConfig).buildClient();
+            mGraphServiceClient = GraphServiceClient.fromConfig(clientConfig);
         }
-
+// GraphServiceClient.Builder().fromConfig(clientConfig).buildClient();
```

Comment:
Lets remove this line if we can since it is not being used.

Reply:
Done. Thanks.